docs/scos/dev/migration-concepts/migrate-to-paas/step-3-migrate-non-standard-services.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Step 3: Migrate non-standard services'
+description: To migrate to PaaS, you need to migrate non-standard services.
+template: howto-guide-template
+---
+
+Spryker comes pre-configured with the following services by default:
+
+* RabbitMQ
+* ElasticSearch
+* Redis
+* MariaDB
+* Jenkins
+
+After you have [upgraded the PHP version](/docs/scos/dev/migration-concepts/migrate-to-paas/step-2-upgrade-the-php-version.html), you can migrate these services without significant challenges, however, make sure to validate their versions. If your project uses components beyond the standard configuration, such as Kafka, we highly recommend to conduct thorough investigation.
```

Comment:
"make sure to validate their versions" - are there any recommendations as to what versions should these services have?
"we highly recommend to conduct thorough investigation" - what specific investigation do you mean here?